src/omero/plugins/duplicate.py

```diff
@@ -51,7 +46,23 @@
     # that would have been duplicated
     omero duplicate Dataset:53 --dry-run --report
 
+    # Duplicate a project with its datasets but not their images
+    omero duplicate Project:15 --ignore-classes=DatasetImageLink
+    # Duplicate a project with the original images linked from its datasets
+    omero duplicate Project:15 --reference-classes=Image
 """
+        "    # Duplicate a project, linking to the original annotations "
+        "except for duplicating the comments and ratings\n"
+        "    omero duplicate Project:15 --reference-classes=Annotation "
+        "--duplicate-classes=CommentAnnotation,LongAnnotation\n"
+        """
+Group permissions can prevent a duplicated link from referencing another
```

Comment:
I'm skeptical that this paragraph is going to be understandable by most. Is there any chance of detecting this particular error and _then_ saying, "you will need to modify your parameters"?

Reply:
Or perhaps at least, I'd begin with a more friendly text describing things in general, and then prefix this paragraph  with Warning/Details/Notice etc.

Reply:
https://trello.com/c/F4cAE0k3/358-graphexception-subclasses may be the best approach there though it'd also require some more analysis to figure what else can cause the same condition.